apps/assistants/forms.py

```diff
@@ -64,6 +58,9 @@ def __init__(self, request, *args, **kwargs):
         self.fields["builtin_tools"].widget.attrs = {
             "x-model.fill": "builtinTools",
         }
+        self.fields["instructions"].help_text = render_help_with_link(
+            INSTRUCTIONS_HELP_TEXT, "/concepts/prompt_variables/"
```

Comment:
How about the link paths to the `DOCUMENTATION_LINKS` variable?

Reply:
Can you explain the rationale behind the `DOCUMENTATION_LINKS` setting? It is just so that all the docs links are centralised?

Reply:
Pretty much. Since it's external links that may change, I feel it would make it just that little bit easier to find and update the previous one. I don't feel very strong about this preference, so welcome to ignore it